README.md

```diff
@@ -18,27 +34,56 @@ The community calendar contains all the upcoming OpenFeature meetings and events
 - [iCal](https://calendar.google.com/calendar/ical/0ua7i1hiv5dh18b27toah63644%40group.calendar.google.com/public/basic.ics)
 
-Notes and recordings from previous meetings can be found below:
 
-- [Meeting notes](https://docs.google.com/document/d/1pp6t2giTcdEdVAri_2B1Z6Mv8mHhvtZT1AmkPV9K7xQ/edit?usp=sharing)
-- [Recordings](https://www.youtube.com/channel/UCXSFt-dT2HORGXz1-ksxtxw)
+### Roadmap
 
-## Slack
+The project roadmap is available [here](https://github.com/orgs/open-feature/projects/1).
+New initiatives require an accepted [OpenFeature Enhancement Proposal (OFEP)](https://github.com/open-feature/ofep). A community decision will then be made as documented in [Decision Making](./governance-charter.md#decision-making).
 
-You can find us in the `#openfeature` channel on the CNCF Slack Workspace. You can join the CNCF Slack Workspace by registering [here](https://slack.cncf.io).
+### Mailing Lists 
+
+TODO
+
+### License
+
+All OpenFeature projects are shipped under the permissive [Apache License v2](./LICENSE) as CNCF IP Policy dictates. This blog post explains the reasoning behind choosing this license. Refer to CONTRIBUTING.md for details on code attribution.
 
-## Community
+### Logos and Brand Guide
+
+The OpenFeature logos and brand guide can be found in the [CNCF artwork repository](https://github.com/cncf/artwork/tree/master/projects/openfeature). In addition, a Google Slides template is also available.
+
+### Social Media 
 
 Follow-us on social media and help us to spread the word!
 Please use the `#openfeature` hashtag or mention our accounts when you share the content.
 
 - Twitter: [@openfeature](https://twitter.com/openfeature)
 - LinkedIn: [OpenFeature](https://www.linkedin.com/company/openfeature/)
 
-## Roadmap
 
-The project roadmap is available [here](https://github.com/orgs/open-feature/projects/1).
-New initiatives require an accepted [OpenFeature Enhancement Proposal (OFEP)](https://github.com/open-feature/ofep). A community decision will then be made as documented in [Decision Making](./governance-charter.md#decision-making).
+### Special Interest Groups
+
+You can find us in the `#openfeature` channel on the CNCF Slack Workspace. You can join the CNCF Slack Workspace by registering [here](https://slack.cncf.io).
+
+Notes and recordings from previous meetings can be found below:
+
+- [Meeting notes](https://docs.google.com/document/d/1pp6t2giTcdEdVAri_2B1Z6Mv8mHhvtZT1AmkPV9K7xQ/edit?usp=sharing)
+- [Recordings](https://www.youtube.com/channel/UCXSFt-dT2HORGXz1-ksxtxw)
+
+| Name | Meeting Time | Meeting Notes | Discussions |
+| ---- | ------------ | ------------- | ----------- |
+|      |              |               |             |    
+
+### Implementation SIGs
+
+You can find us in the `#openfeature` channel on the CNCF Slack Workspace. You can join the CNCF Slack Workspace by registering [here](https://slack.cncf.io).
+
+TODO:
```

Comment:
Please create an issue and remove this section.

Reply:
done